crates/unified-bridge/src/imported_bridge_exit.rs

```diff
@@ -349,6 +349,21 @@ impl ImportedBridgeExit {
     }
 }
 
+impl ImportedBridgeExit {
+    pub fn valid_claim(&self) -> bool {
+        match &self.claim_data {
+            Claim::Mainnet(claim) => {
+                claim.l1_leaf.inner.global_exit_root
+                    == keccak256_combine([claim.l1_leaf.mer, claim.l1_leaf.rer])
+            }
+            Claim::Rollup(claim) => {
+                claim.l1_leaf.inner.global_exit_root
+                    == keccak256_combine([claim.l1_leaf.mer, claim.l1_leaf.rer])
+            }
+        }
```

Comment:
Nit: just remove duplication: `Claim::Mainnet(claim)  | Claim::Rollup(claim)`

Reply:
`claim` is not the same type unfortunately.